articles/building-a-simple-cryptocurrency-blockchain/index.md

```diff
@@ -0,0 +1,187 @@
+## Building Simple cryptocurrency system using Node.js
+
+In this tutorial, we are going to demonstrate how to build a simple cryptocurrency system, called thecoin.
+It is a simple implementation of the of decentralized digital money, based on blockchain technology. 
+### Take away
+1. How blockchain store data
+2. How blockchain data are immutable
+3. How to implement JavaScript libary in Node.js 
+
+
+![Image title](engineering-education/building-a-simple-cryptocurrency-blockchain/trade.jpg)
+
+ ### We are going to implement the concepts of :
+
+1. [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+2. [Node.js](https://nodejs.org/en/docs/)
+
+## Prerequisite
+To follow this tutorial successfully, you’ll need to have the following:
+
+- Node.js Installed in your machine
+
+- Code editor for my case I use VSCode
+
+
+### What is Blockchain?
+![Bitcoin Image](engineering-education/building-a-simple-cryptocurrency-blockchain/bitcoin.jpg)
+Digital currencies such as Bitcoin, Ethereum are powered and adopted with powerful technology **the Blockchain**. It uses Cryptography to securely connect and maintain a list of records growing continuously known as **Blocks**
+![ethereum](engineering-education/building-a-simple-cryptocurrency-blockchain/ethereum.jpg)
+**Blockchain** as the name states it's a block of transactions data growing increasingly to create a chain of transaction occurrences. Valid transaction data are logged into the blockchain network following the peer-to-peer rule laid down by participants.
```

Comment:
Could you explain more briefly? This paragraph can be understood by someone, who already knows what a blockchain is. But, you must assume the perspective of the reader who is beginner.

As a suggestion,
- You could relate blockchain with something else, and give an analogy
- You could talk about its benefits
- Talk about decentralization
- Talk about the use of blockchain in the present world
- Describe about the security system and its peer-peer network.
- Explain its various steps of working.

In fact, you could have separate sections for each of the above topics.

Reply:
Hi @srishilesh  Just provided it on the introduction part